sdk/search/search-documents/README.md

```diff
@@ -549,6 +549,32 @@ async function main() {
 main();
 ```
 
+#### Get a list of existing datasources in the service
+```js
+const { SearchServiceClient, AzureKeyCredential } = require("@azure/search-documents");
+
+const client = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
+
+async function main() {
+  let listOfDataSources = await client.listDataSources();
```

Comment:
nit: both this and the the iterator in the for loop can be `const` instead

Reply:
My bad... Copied from a previous case and missed it. Will change in the next commit